transcom/mymove
# INT B-19780 Merge Fix Follow Up

## [B-19780](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19780)

## Summary

Re-adds code that was lost in an unrelated merge. Missing code made the origin duty location column in the export file for the counseling queue and PPM closeout queue blank.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the office app as a services counselor.
2. Export your main queue.
3. Verify the exported file contains data in the origin duty location column.
4. Repeat for the PPM Closeout Queue.

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
